app/controllers/users_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```

Comment:
Please remove commented out code. If you need to review a previous version of your code, use git history.

Reply:
Thank you for your feedback. We have made the requested change. :)